### PR TITLE
added the action dimension and visit dimension in all supported method

### PR DESCRIPTION
**Pull request by kaushik-patel-us**

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -187,7 +278,18 @@ export default function App() {
             style={styles.button}
             onPress={() => {
               trackMediaEvent({ siteId: "siteId", mediaId: Date.now.toString(), mediaTitle: "video media play track", playerName: "test 08", mediaType: MediaType.VIDEO, mediaResource: "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4", mediaStatus: "100", mediaLength: "100", mediaFullScreen: "1", mediaHeight: "720", mediaWidth: "1080", mediaProgress: "100",
-              dimensions:[{key:"1",value: "cf7fad2e-fae4-4c49-9924-ad9a2a7c50de"}]
+              actionDimensions:{
```
Comment (bhavesh-lw) [inline example/src/App.tsx:281]: @kaushik-patel-us : Please verify key over here. Should be dimentions

File: README.md [hunk 2]
```diff
@@ -98,30 +102,68 @@ startSession()
 
 ### trackScreen()
 
-The trackScreen method is used to track screen views within a React Native application. It will take `screen name` and `title` parameter.
+The trackScreen method is used to track screen views within a React Native application. It will take `screen name`,`title` with the optional parameter `actionDimensions`.
```
Comment (bhavesh-lw) [inline README.md:105]: @kaushik-patel-us : verify parameter name

**Review by husenLogicwind — APPROVED** [review]
pass

**Review by nikesh8 — APPROVED** [review]
Reviewed

**Review by husenLogicwind — APPROVED** [review]

**Review by bhavesh-lw — APPROVED** [review]
Approved

**Pull request merged**